health-services/project-factory/src/server/generateFlowClasses/user-generateClass.ts

```
@@ -0,0 +1,164 @@
+        }, {});
+
+        const boundaryCodes: any = new Set(boundaries.map((boundary: any) => boundary.code));
+
+        await populateBoundariesRecursively(
+            boundaryRelationshipResponse?.TenantBoundary?.[0]?.boundary?.[0],
+            boundaries,
+            boundaryChildren[boundaryRelationshipResponse?.TenantBoundary?.[0]?.boundary?.[0]?.code],
+            boundaryCodes,
+            boundaryChildren
+        );
+        const structuredBoundaries = this.structureBoundaries(boundaries, campaignDetails?.hierarchyType, localizationMap);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)